flutter_local_notifications/example/lib/main.dart

```diff
@@ -716,15 +854,22 @@ class _HomePageState extends State<HomePage> {
         IOSNotificationDetails(sound: 'slow_spring_board.aiff');
     const MacOSNotificationDetails macOSPlatformChannelSpecifics =
         MacOSNotificationDetails(sound: 'slow_spring_board.aiff');
-    const NotificationDetails platformChannelSpecifics = NotificationDetails(
-        android: androidPlatformChannelSpecifics,
-        iOS: iOSPlatformChannelSpecifics,
-        macOS: macOSPlatformChannelSpecifics);
+    final LinuxNotificationDetails linuxPlatformChannelSpecifics =
```

Comment:
@proninyaroslav I left a comment on this elsewhere that I think may have been lost but wanted to flag that I found this sound wouldn't play when I tried on an Ubuntu VM. Do you have the same problem and are you able to help with this?

Reply:
Unfortunately this is not standardized. You can get different behavior in different DEs. For example, on GNOME (which is used by Ubuntu) I was unable to achieve mp3 playback, it was causing the system to crash. I checked the `/usr/share/sounds/freedesktop` directory and it turns out that the system uses the Ogg format for various system sounds. Therefore, you can use `ThemeLinuxSound` to specify the name of the system sound, or try using `.oga` audio files. I think that in the example it's worth trying to replace the mp3 sound  with Ogg format.

Reply:
Thanks for the reply. The ogg format sounds like a good idea 🙂